_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -471,87 +460,78 @@ namespace ExtBuffer
 
     #undef EXTBUF
 
-    #define _CopyPar(dst, src, PAR) dst.PAR = src.PAR;
-    #define _CopyPar1(PAR) _CopyPar(buf_dst, buf_src, PAR);
-    #define _CopyStruct1(PAR) Copy(buf_dst.PAR, buf_src.PAR);
-    #define _NO_CHECK()  buf_dst = buf_src;
-
-    template <class T> void CopySupportedParams(T& buf_dst, T& buf_src)
```

Comment:
I rechecked the code. We have to restore this one because of possible overloadings